io/disk/lvsetup.py.data/lvsetup.yaml

```diff
@@ -0,0 +1,31 @@
+setup:
```

Comment:
Just added all possible inputs for the test as a sample yaml file. I think the above simpler form will be fine. I will change it.